package.json

```diff
@@ -60,13 +61,15 @@
     "pre-commit": "^0.0.9",
     "tape": "^3.0.3",
     "tape-cluster": "2.1.0",
-    "tchannel": "^3.6.13",
     "tcurl": "^4.11.1",
     "timer-shim": "^0.3.0",
     "tryit": "^1.0.1",
     "uber-licence": "^2.0.1",
     "winston": "^1.0.1"
   },
+  "peerDependencies": {
+    "tchannel": "^3.6.13"
```

Comment:
Since a `tchannel`-instance is passed into `ringpop` during construction, it should be listed as a peer dependency so we can explicitly state our supported version when a library is using ringpop as a dependency.